vignettes/FDZ_Data_Cleaning.Rmd

```diff
@@ -80,8 +80,8 @@ extractMeta(dat2, vars = c("school"))
 ```
 The previous function had already tagged two of the missings correctly, but the value label `-98` with the value label `unclear answer` was not used. Now, it is contained in the specified range and therefore tagged as a missing.
 
-## Define missing values by hand
-At times, only individual missings need to be tagged. `changeMissings()` tags missing values as missings by specifying them under `value` and marks them as `miss` under `missings.`
+## Defining missing values by hand
+At times, only individual missings need to be tagged. `changeMissings()` tags missing values as missings by specifying them under `value` and marks them as `miss` under `missings.` This is also possible for multiple variables at the same time.
```

Comment:
LGTM

Nit: 
Would there be other variables besides "info" for which setting these missing tags make sense? To illustrate this specific functionality?

Reply:
The variable “subjfav” also contains the code -97, which could be tagged as a missing code at the same time as “info”. However, the two missing codes -98 and -99 are then added to the variable, which were not there before. It would be a question of style, I think. I can also simply add a suitable variable to the example data set or insert two more missing codes (-98 and -99) in “subjfav” so it harmonizes well.

Reply:
yes, no problem!